src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {
+
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Consider moving the `ColorSupplier` to a class-level field, as creating a new instance every time the method is called is not efficient when a single instance can be reused.